### DISCUSSION TITLE
When i go full screen i need to manually hide the top bar. How to automate it?{look trough the image to understand the problem} Many thanks

### DISCUSSION THREAD

**Discussion by ghost** [mpv-player/mpv#15605] [Q&A] [answered]
![1](https://github.com/user-attachments/assets/89d7066c-44f5-4c69-989d-aa53088dc3a4)
![2](https://github.com/user-attachments/assets/2749f4e3-10db-4c9c-9094-3ddce4f49542)
![3](https://github.com/user-attachments/assets/6d9856fd-1342-4625-8e95-d8d9985c5ba3)
![4](https://github.com/user-attachments/assets/a5567874-eeef-4754-8b0c-468c86ebecd1)

**Comment by ghost** [comment]
Previously on linux it was ok. But in windows i am having this issue. I am using uosc script.

**Comment by Nyaa-rlathotep** [comment, marked as answer]
Just to double-check, are you actually telling it to go fullscreen that first time with the "F" key or anything like that? It looks like the window is just maximized, which is totally different than telling it to go fullscreen.

**Comment by ghost** [comment]
Thanks for the shortcut.
I pressed the maximize button. In fedora pressing that would go full screen. I totally expected it to behave like that.
